VS_Project/Pokerapp/Pokerapp/Tableview.cshtml

```diff
@@ -81,24 +83,21 @@
             Response.Redirect("~/" + Url);
         }*@
     }
-    <h1>Table @table</h1>
-    <br />
-    <div>
-        @foreach (var a in db.Query("SELECT TOP (@0) Name FROM Players WHERE Room_Code = @1 AND Tbl_nr = @2 ORDER BY Name ASC", eerste, roomCode, table))
-        {
-            @a.Name
-            <img src="~/stoel.png" name="chair" />
-        }
-    </div><br />
-    <div>
-        <img src="~/tafel.png" name="pokertable" />
-    </div>
-    <br />
-    <div>
-        @foreach (var a in db.Query("SELECT TOP (@2) Name FROM Players WHERE Room_Code = @0 AND Tbl_nr = @1 ORDER BY Name DESC", roomCode, table, tweede))
-        {
-            <img src="~/stoel.png" name="chair" />
-            @a.Name
-        }
-    </div>
+<h1>Table @table</h1>
+<br />
+<div>
+    @foreach (var a in db.Query("SELECT TOP (@0) Name FROM Players WHERE Room_Code = @1 AND Tbl_nr = @2 ORDER BY Name ASC", eerste, urlcode, table))
```

Comment:
Waar haal jij urlcode vandaan? Er wordt bovenaan een roomCode gedefinieerd, deze word uit de URL gehaald.
Het lijkt er niet op dat je urlcode ergens gedefinieerd staat, heb je je code wel getest voordat je het pushte? Want dit zou in theorie niet moeten runnen. Of in ieder geval zou je query elke keer een null geven of je krijgt een NullReferenceException.